packages/core/src/types.ts

```diff
@@ -250,11 +250,7 @@ export function indexSearchParameterBundle(bundle: Bundle<SearchParameter>): voi
  * @see {@link IndexedStructureDefinition} for more details on indexed StructureDefinitions.
  */
 export function indexSearchParameter(searchParam: SearchParameter): void {
-  if (!searchParam.base) {
```

Comment:
Why this change?  If `base` is `undefined`, then the function is a no-op, yeah?

Reply:
I kept it as a no-op, just streamlined the undefined check so it would instead iterate over an empty array